README.md

```diff
@@ -46,11 +48,12 @@ Before deploying the bundle you may want to create a dedicated model for Tempo c
 juju add-model tempo
 ```
 
-### Monolithic deployment
-The bundle deploys Tempo in its [monolithic mode](https://grafana.com/docs/tempo/latest/setup/deployment/#monolithic-mode), which runs all required components in a single process and is the default mode of operation. Monolithic mode is the simplest way to deploy Grafana Tempo and is useful if you want to get started quickly or want to work with Grafana Tempo in a development environment.
+### Scaling monolithic deployment
```

Comment:
I wonder if we should keep calling this the 'monolithic mode', to avoid suggesting that we also have a non-scaling one.
Much like we internally remap `all` to `scaling-single-binary`.
Perhaps add a note that the official docs call it differently, but to us this is just "the monolithic mode"? Or will that be just confusing?
@simskij WDYT?

Reply:
IMO, I think it would be confusing. I think we clearly say that we only have 3 modes of operation and we just mention what is that mean in the official docs.

Reply:
I think I'm with Michael on this one.